contracts/StableJoeStaking.sol

```diff
@@ -66,6 +67,9 @@ contract StableJoeStaking is Initializable, OwnableUpgradeable {
     /// @dev Info of each user that stakes JOE
     mapping(address => UserInfo) private userInfo;
 
+    /// @dev Smol Joes contract address
+    IERC721Upgradeable smolJoes;
```

Comment:
NIT: Mark this variable explicitly as public (default if not specified) to match the style of other variables.